django/core/validators.py

```diff
@@ -301,7 +304,7 @@ def ip_address_validators(protocol, unpack_ipv4):
 
 
 def int_list_validator(sep=',', message=None, code='invalid', allow_negative=False):
-    regexp = _lazy_re_compile(r'^%(neg)s\d+(?:%(sep)s%(neg)s\d+)*\Z' % {
```

Comment:
Given that `integer_validator` stays at `\d` I wonder if this one shouldn't stay with `\d` as well. Semantically there doesn't seem much of a diff to me between a single integer and a list of integers that would explain the difference in the regex.

Reply:
The main difference is that `integer_validator` is not used internally. `int_list_validator` is used in (deprecated) `CommaSeparatedIntegerField` and it's recommended as a validator for `CharField` that can be used instead. Unfortunately, values are not cast to `int()`, so non-ASCII digits can cause issues.

We can change both :thinking:

Reply:
@apollo13 What do you think? We can change both or neither.

Reply:
OK, will revert :+1: